src/core/automation.py

```
@@ -327,6 +331,61 @@ def request_accessibility_permissions(self) -> None:
+            return None
+        
+        try:
+            from typing import Dict, Any
+            
+            # システムのウィンドウリストを取得
+            window_list = CGWindowListCopyWindowInfo(
+                kCGWindowListExcludeDesktopElements,
+                kCGNullWindowID
+            )
+            
+            kindle_bundle_id = "com.amazon.Lassen"
```

Ruff (0.12.2)

Local variable `kindle_bundle_id` is assigned to but never used

Remove assignment to unused variable `kindle_bundle_id`

(F841)